### PR TITLE
refactor: refactor and update util to use new log formatter setting

### PR DESCRIPTION
**Pull request by blackcat-118**
## Description
- Refactor logger to free5GC logger format
- Add CI actions 
- Update module package

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,26 +1,54 @@
...
-	git.cs.nctu.edu.tw/calee/sctp v1.1.0
-	github.com/antonfisher/nested-logrus-formatter v1.3.1
 	github.com/asaskevich/govalidator v0.0.0-20210307081110-f21760c49a8d
-	github.com/free5gc/aper v1.0.5-0.20230614030933-c73735898582
+	github.com/free5gc/aper v1.0.5
 	github.com/free5gc/nas v1.0.7
-	github.com/free5gc/ngap v1.0.6
-	github.com/free5gc/util v1.0.4
+	github.com/free5gc/ngap v1.0.9-0.20240414165820-453b0aa37228
+	github.com/free5gc/sctp v1.0.0
```
Comment (andy89923) [inline go.mod:10]: change to v1.0.1

File: internal/ngap/handler/handler.go [hunk 2]
```diff
@@ -6,7 +6,7 @@ import (
 	"net"
 	// "time"
```
Comment (andy89923) [inline internal/ngap/handler/handler.go:7]: if not use, remove it

File: pkg/context/context.go [hunk 3]
```diff
@@ -4,18 +4,20 @@ import (
...
+	"github.com/free5gc/sctp"
 	"github.com/sirupsen/logrus"
 	gtpv1 "github.com/wmnsk/go-gtp/gtpv1"
 	"golang.org/x/net/ipv4"
 
-	"github.com/free5gc/tngf/internal/logger"
 	"github.com/free5gc/ngap/ngapType"
+	"github.com/free5gc/tngf/internal/logger"
 	"github.com/free5gc/util/idgenerator"
 	// "fmt"
```
Comment (andy89923) [inline pkg/context/context.go:22]: remove unused import

File: pkg/service/init.go [hunk 4]
```diff
@@ -56,116 +56,66 @@ var cliCmd = []cli.Flag{
 	},
```
Comment (andy89923) [inline pkg/service/init.go:56]: if not used, remove it.

**Review by andy89923 — CHANGES_REQUESTED** [review]
- Add Github CI Actions 
- Add linter

File: README.md [hunk 5]
```diff
@@ -1,93 +0,0 @@
-# tngf
-
```
Comment (ianchen0119) [inline README.md:2]: @blackcat-118 

Why remove the README file?

Comment (andy89923) [reply README.md:2]: All the content was generated by GitLab.

**Pull request closed without merging**